app/src/sharedTest/java/org/oppia/app/topic/conceptcard/ConceptCardFragmentTest.kt

```diff
@@ -39,13 +40,13 @@ class ConceptCardFragmentTest {
   @Test
   fun testConceptCardFragment_openDialogFragmentWithSkill1_explanationIsDisplayed() {
     onView(withId(R.id.open_dialog_1)).perform(click())
-    onView(withId(R.id.explanation)).check(matches(withText("Explanation with <b>rich text</b>.")))
+    onView(withId(R.id.explanation)).check(matches(withText("Explanation with rich text.")))
   }
 
   @Test
   fun testConceptCardFragment_openDialogFragmentWithSkill1_workedExamplesAreDisplayed() {
     onView(withId(R.id.open_dialog_1)).perform(click())
-    onView(withText("Worked example with <i>rich text</i>.")).check(matches(isDisplayed()))
+    onView(atPositionOnView(R.id.worked_examples, 0, R.id.example)).check(matches(withText("Worked example with rich text.")))
```

Comment:
Can you also check that we have a rich text span present in the spannable? That's one way to verify that the rich-text conversion is correct. As it stands, I *think* these tests will pass even if you remove the HTML parsing, which means we aren't actually adding coverage for that new behavior that's being implemented in this PR.

Reply:
I wrote a matcher that calls getSpan on the view's text to see if it contains any spans to test specially for rich text.